src/cocoa/toga_cocoa/widgets/imageview.py

```diff
@@ -14,7 +14,6 @@ class ImageView(Widget):
 
     def create(self):
         self.native = NSImageView.alloc().init()
-        self.native.interface = self.interface
```

Comment:
As above - any reason to drop the  consistency here? This one is a little more significant than Divider, because users might actually need to do interact with the widget.

Reply:
Answer as above.